docs/guide/acquiring_data/fido.rst

```diff
@@ -107,8 +107,24 @@ Downloading data
 Once you have located your files via a ```Fido.search```, you can download
 them via ```Fido.fetch```::
 
-    >>> downloaded_files = Fido.fetch(result)
+    >>> downloaded_files = Fido.fetch(results)
```

Comment:
Was it not decided 'fetch'es are changing to `get`s? I may have misremembered this.